unitTests/ui/MissingTermsTest.ts

```diff
@@ -0,0 +1,217 @@
+import * as Mock from '../MockEnvironment';
+import { MissingTerms } from '../../src/ui/MissingTerm/MissingTerms';
+import { $$ } from '../../src/Core';
+import { FakeResults } from '../Fake';
+
+export function MissingTermsTest() {
+  describe('MissingTerm', () => {
+    const mockComponent = (query: string, option = {}) => {
+      return Mock.advancedResultComponentSetup<MissingTerms>(
+        MissingTerms,
+        fakeResult,
+        new Mock.AdvancedComponentSetupOptions(null, option, (env: Mock.MockEnvironmentBuilder): Mock.MockEnvironmentBuilder => {
+          env.withLiveQueryStateModel().queryStateModel.set('q', query);
+          return env;
+        })
+      );
+    };
+
+    let test: Mock.IBasicComponentSetup<MissingTerms>;
+    let fakeResult = FakeResults.createFakeResult();
+    describe('exposes options', () => {
+      describe("when the 'clickable' option is set to", () => {
+        let missingTermsClickableSpy: jasmine.Spy;
```

Comment:
I guess we could configure the prettifier a bit more, but just in general, you can add spacing between blocks, it's kind of inconsistent.